.github/workflows/csharp-app-release-with-docs.yml

```diff
@@ -64,7 +64,7 @@ on:
 jobs:
   release-checks:
     runs-on: ubuntu-latest
-    container: zepben/pipeline-basic:5.7.5
+    container: zepben/pipeline-basic
```

Comment:
DO NOT MERGE! This will be merged after new pipeline containers are built

- https://github.com/zepben/ci-utils/pull/19